src/Form/Control/Calendar.php

```diff
@@ -33,62 +31,30 @@ public function setOption($name, $value)
         $this->options[$name] = $value;
     }
 
-    /**
-     * Set first day of week globally.
-     */
-    public static function setFirstDayOfWeek(App $app, int $day)
-    {
-        $app->html->js(true, (new JsExpression('flatpickr.l10ns.default.firstDayOfWeek = [day]', ['day' => $day])));
-    }
-
-    /**
-     * Load flatpickr locale file.
-     * Pass it has an option when adding Calendar input.
-     *  Form\Control\Calendar::requireLocale($app, 'fr');
-     *  $form->getControl('date')->options['locale'] = 'fr';.
-     */
-    public static function requireLocale(App $app, string $locale)
```

Comment:
Wy not keep this static method in order to easily load locale setting within a multi language environment?

Reply:
because "multi language environment" in a single app/render is non-sense :)

Reply:
@ibelar does it make sense, ok to merge?